db.go

```diff
@@ -75,12 +75,26 @@ func (self *DB) Query(sql string, vars map[string]any) (any, error) {
 	return self.send("query", sql, vars)
 }
 
+func (self *DB) SchemalessSelect(what string) ([]map[string]interface{}, error) {
```

Comment:
instead of the many `interface{}`, you can use `any` because this is go 1.18

Reply:
Changed